src/iam/config/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2025 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME config)
+
+# ######################################################################################################################
+# Sources
+# ######################################################################################################################
+
+set(SOURCES config.cpp)
+
+# ######################################################################################################################
+# Libraries
+# ######################################################################################################################
+
+set(LIBRARIES aoscommon ${COMMON_NAMESPACE}::utils Poco::JSON)
```

Comment:
add local modules first then core lib then 3rd parties. Update in all modules and tests.